build-system/tasks/runtime-test/helpers-unit.js

```diff
@@ -119,15 +111,16 @@ function getJsFilesFor(cssFile, cssJsFileMap) {
     });
     jsFilesInDir.forEach((jsFile) => {
       const jsFilePath = `${cssFileDir}/${jsFile}`;
-      if (getImports(jsFilePath).includes(cssJsFileMap[cssFile])) {
+      const cssFilePath = path.resolve(cssJsFileMap[cssFile]);
+      if (areAnyDependencies(jsFile, [cssFilePath])) {
         jsFiles.push(jsFilePath);
       }
     });
   }
   return jsFiles;
 }
 
-function getUnitTestsToRun() {
+function getUnitTestsToRun(unitTestPaths = testConfig.unitTestPaths) {
```

Comment:
Since there's only one call site for this function that is passing in `unitTestPaths`, is the default value necessary?